### PR TITLE
[WIP] Add unit tests for grant application pages

### PR DESCRIPTION
**Pull request by monishdeb**

### CODE REVIEW

File: tests/phpunit/CRM/Grant/Form/Grant/MainTest.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ *    rollback automatically -- as long as you don't manipulate schema or truncate tables.
+ *    If this test needs to manipulate schema or truncate tables, then either:
+ *       a. Do all that using setupHeadless() and Civi\Test.
+ *       b. Disable TransactionalInterface, and handle all setup/teardown yourself.
+ *
+ * @group headless
+ */
+class CRM_Grant_Form_Grant_MainTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
+
+  public function setUpHeadless() {
```
Comment (seamuslee001) [inline tests/phpunit/CRM/Grant/Form/Grant/MainTest.php:24]: Suggest putting in before this use \Civi\Test\Api3TestTrait

File: tests/phpunit/CRM/Grant/Form/GrantPageTest.php [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+ *    rollback automatically -- as long as you don't manipulate schema or truncate tables.
+ *    If this test needs to manipulate schema or truncate tables, then either:
+ *       a. Do all that using setupHeadless() and Civi\Test.
+ *       b. Disable TransactionalInterface, and handle all setup/teardown yourself.
+ *
+ * @group headless
+ */
+class CRM_Grant_Form_GrantPageTest extends \PHPUnit\Framework\TestCase implements HeadlessInterface, HookInterface, TransactionalInterface {
+
+  public function setUpHeadless() {
```
Comment (seamuslee001) [inline tests/phpunit/CRM/Grant/Form/GrantPageTest.php:23]: Suggest adding before this \Civi\Test\Api3TestTrait

Comment (monishdeb) [reply tests/phpunit/CRM/Grant/Form/GrantPageTest.php:23]: agree. thanks for pointing that :)

File: CRM/Grant/Form/Grant/Confirm.php [hunk 3]
```diff
@@ -542,6 +542,10 @@ public function postProcess() {
 
     $grantParams = $this->_params;
 
+    CRM_Core_Error::debug_var('$grantParams', $grantParams);
+    CRM_Core_Error::debug_var('_values', $this->_values);
+    CRM_Core_Error::debug_var('$fieldTypes', $fieldTypes);
+
```
Comment (Edzelopez) [inline CRM/Grant/Form/Grant/Confirm.php:548]: Do these need to be added?

**Comment by monishdeb** [comment]
Moved to https://lab.civicrm.org/extensions/grantapplications/-/merge_requests/164

**Pull request closed without merging**